frontend/packages/ux-editor/src/components/config/editModal/EditOptions/OptionTabs/EditTab/EditTab.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { StudioDeleteButton } from '@studio/components';
+import { EditManualOptionsWithEditor } from './EditManualOptionsWithEditor';
+import { OptionListSelector } from './OptionListSelector';
+import { OptionListUploader } from './OptionListUploader';
+import { OptionListEditor } from './/OptionListEditor';
+import type { IGenericEditComponent } from '../../../../componentConfig';
+import type { SelectionComponentType } from '../../../../../../types/FormComponent';
+import classes from './EditTab.module.css';
+
+type EditOptionChoiceProps = Pick<
+  IGenericEditComponent<SelectionComponentType>,
+  'component' | 'handleComponentChange'
+>;
+
+export function EditTab({
+  component,
+  handleComponentChange,
+}: EditOptionChoiceProps): React.ReactElement {
+  const componentHasOptionList: boolean = !!component.optionsId || !!component.options;
+  const [chosenOption, setChosenOption] = useState<boolean>(componentHasOptionList);
+
+  return (
+    <>
+      {chosenOption ? (
+        <SelectedOptionList
+          setChosenOption={setChosenOption}
+          component={component}
+          handleComponentChange={handleComponentChange}
+        />
+      ) : (
+        <div className={classes.optionButtons}>
+          <EditManualOptionsWithEditor
+            setChosenOption={setChosenOption}
+            component={component}
+            handleComponentChange={handleComponentChange}
+          />
+          <OptionListSelector
+            setChosenOption={setChosenOption}
+            component={component}
+            handleComponentChange={handleComponentChange}
+          />
+          <OptionListUploader
+            setChosenOption={setChosenOption}
+            component={component}
+            handleComponentChange={handleComponentChange}
+          />
+        </div>
+      )}
+    </>
+  );
+}
+
+type SelectedOptionListProps = {
+  setChosenOption: (value: boolean) => void;
+} & Pick<IGenericEditComponent<SelectionComponentType>, 'component' | 'handleComponentChange'>;
+
+function SelectedOptionList({
+  setChosenOption,
+  component,
+  handleComponentChange,
+}: SelectedOptionListProps) {
+  const { t } = useTranslation();
+
+  const handleClick = () => {
```

Comment:
Maybe rename this to 
```suggestion
  const handleDelete = () => {
```